configuration/cdap-env.xml

```diff
@@ -172,6 +172,7 @@ export AUTH_JAVA_HEAPMAX="-Xmx{{cdap_auth_heapsize}}"
 export KAFKA_JAVA_HEAPMAX="-Xmx{{cdap_kafka_heapsize}}"
 export MASTER_JAVA_HEAPMAX="-Xmx{{cdap_master_heapsize}}"
 export ROUTER_JAVA_HEAPMAX="-Xmx{{cdap_router_heapsize}}"
+export SPARK_HOME="/usr/hdp/{{hdp_version}}/spark"
```

Comment:
A small error: line 182:
```
     </value>
```
That little bit of leading whitespace causes a problem in the Ambari UI: clicking the "Set Recommended" button loads this, and gives an error about how there can be no trailing whitespace. Solution is probably one of either 
```
</value>
```
or
```
export TEZ_CONF_DIR="/etc/tez/conf"</value>
```